packages/govern-console/src/containers/CreateDao/CreateDao.tsx

```diff
@@ -10,16 +10,15 @@ import { CreateDaoProvider } from './utils/CreateDaoContextProvider';
 import PageContent from 'components/PageContent/PageContent';
 import { PageName } from 'utils/HelpText';
 import ReviewCard from './components/ReviewCard';
+import scrollToTop from '../../utils/scrollToId';
```

Comment:
@Rekard0 could you import this without relative path ? the same kind of way as `utils/HelpText` is imported above.

Reply:
done